vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -34,8 +34,9 @@ az resource create --id $disk_resource_id  --is-full-object --location $LOCATION
 }"
 # shellcheck disable=SC2102
 sas=$(az disk grant-access --ids $disk_resource_id --duration-in-seconds 3600 --query [accessSas] -o tsv)
+echo "Converting $sig_resource_id to ${CLASSIC_BLOB}/${GEN2_CAPTURED_SIG_VERSION}.vhd"
 
-azcopy-preview copy "${sas}" "${CLASSIC_BLOB}/${GEN2_CAPTURED_SIG_VERSION}.vhd${CLASSIC_SAS_TOKEN}" --recursive=true
+# azcopy-preview copy "${sas}" "${CLASSIC_BLOB}/${GEN2_CAPTURED_SIG_VERSION}.vhd${CLASSIC_SAS_TOKEN}" --recursive=true
```

Comment:
Why this line is commented out? It is required for Linux Gen2 VHD build.

For Linux Gen1 VHD build, packer can capture os disk directly to a vhd blob for publish pipeline to use (e.g. https://vhdbuilderaceastustest.blob.core.windows.net/vhd/aks-1659383205-osDisk.bd14574a-4a0d-4c2b-9e73-ae1d01bbe095.vhd ...)

For Linux Gen2 VHD build, packer cannot capture os disk directly to a vhd blob for publish pipeline to use later. The packer captures the os disk to a managed image, convert the managed image to a SIG (this is for amd64, arm64 is more complext). then this .sh script is called to convert the SIG to a vhd blob for publish pipeline to use later (e.g. https://vhdbuilderaceastustest.blob.core.windows.net/vhd/1.1659383200.4650.vhd ...). if this line is commented out, the contract between the VHD build pipeline and VHD publish pipeline for Gen2 VHDs are broken.

Reply:
Uncommented.